pkg/scheduler/core/generic_scheduler.go

```diff
@@ -373,7 +371,7 @@ func (g *genericScheduler) prioritizeNodes(
 
 	if klog.V(10).Enabled() {
 		for plugin, nodeScoreList := range scoresMap {
-			klog.Infof("Plugin %s scores on %v/%v => %v", plugin, pod.Namespace, pod.Name, nodeScoreList)
+			klog.InfoS("Plugin scored nodes for pod", "pod", klog.KObj(pod), "plugin", plugin, "nodeScoreList", nodeScoreList)
```

Comment:
Extend this to a for loop, and have one line for each node, just like we do for extenders.

Reply:
Okey, I fixed it like this:
if klog.V(10).Enabled() {
	for plugin, nodeScoreList := range scoresMap {
		for _, nodeScore := range nodeScoreList {
			klog.InfoS("Plugin scored node for pod", "pod", klog.KObj(pod), "plugin", plugin, "node", nodeScore.Name, "score", nodeScore.Score)
		}
	}
}